server/src/main/java/org/solarbank/server/ErrorMessage.java

```diff
@@ -0,0 +1,18 @@
+package org.solarbank.server;
+
+import lombok.Getter;
+
+@Getter
+public enum ErrorMessage {
+    BAD_REQUEST("Bad Request"),
+    INTERNAL_SERVER_ERROR("Internal Server Error"),
+    INTERNAL_SERVER_ERROR_DETAILS("An unexpected error occurred. Please try again later."),
+    NO_BINDING_RESULT("Binding result not found"),
```

Comment:
I think this can be removed with the changes to `ControllerExceptionHandler`.

Reply:
removed

Reply:
removed